### ISSUE TITLE
Filter dropdown for Recommendations should not side scroll

### ISSUE THREAD

**Issue by ericksod** [cloud-carbon-footprint/cloud-carbon-footprint#473]
### Preflight Checklist

- [ ] I have read the [Contributing Guidelines](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/blob/trunk/CONTRIBUTING.md) for this project.
- [ ] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details
On the recommendations dashboard, we have a dropdown filter for recommendations. Currently this dropdown filter has both a vertical scroll and a horizontal scroll. Having a horizontal scroll inside of a dropdown is a poor user experience. We would like to fix this.
Does not seem to show on all machines.
- **Operating System:**
  - 
- **Browser:**
  - 

### Expected Behavior
- [ ] When I view any dropdown filter on the recommendations or home page, I see only a vertical scroll - no horizontal scroll

### Actual Behavior
-  recommendations dropdown filter on the recommendations page, shows a horizontal scroll bar
- services dropdown filter shows horizontal scroll bar

![image (5)](https://user-images.githubusercontent.com/8618241/133680143-8945dc11-0d0b-4505-9d2f-0fab4fd0da3c.png)

### To Reproduce
- Go to recommendations dashboard and click on recommendations dropdown

### Additional Information

© 2021 Thoughtworks, Inc.

**Comment by logicalahara** [comment]
Hey @ericksod, I'd like to contribute to this bug.

Will you please assign this issue

**Comment by logicalahara** [comment]
is there a specific branch to work on or should I create my own branch ?

**Comment by ericksod** [comment]
Hi @logicalahara!
Sounds great- I'll add you to the card.
You can just submit a PR once you've come up with a solution, and then we'll review it and merge it.
Let us know if you have further questions, thanks!

**Comment by ericksod** [comment]
Hey @logicalahara!
If you're still working on this, just want to let you know that we've also seen the side scroll issue on the services dropdown (home page and recommendations page), so I've added that to the card.
Basically if the solution could prevent side scrolling on all filter dropdowns, that would be ideal

**Comment by 4upz-tw** [comment]
Hi @logicalahara, were you still working on this? If so, then no rush! Just wanted to check in on the progress and see if you needed any support. Feel free to reach out to myself, or any of us, with any questions or concerns

**Comment by ghost** [comment]
Hi @logicalahara, we're going to remove you for now in case another contributor would like to pick this up. If you end up having time to look at this in the near future, feel free to pick this back up.

**Comment by ericksod** [comment]
It seems like we now have some services cut off on the main emissions page services dropdown, unfortunately.
But the recommendations page looks good.
<img width="352" alt="Screen Shot 2021-11-19 at 9 37 48 AM" src="https://user-images.githubusercontent.com/8618241/142640661-69be2d3f-f4ed-4ff4-b658-38a9e991eeb0.png">
ce

**Comment by charmilahruthika** [comment]
Hey @ericksod , I would like to fix this issue, could you assign this to me.

**Comment by 4upz** [comment]
Hi @charmilahruthika Thanks for volunteering! I've assigned you to this issue so you can begin workin on it. Feel free to reach out if you have any questions or need any support 😃

**Comment by hariomsinha7** [comment]
Hi, I would like to fix this issue. Is this completed or can I work on this ? Thanks.

**Comment by 4upz** [comment]
Hi @hariomsinha7 , I believe that @charmilahruthika is still working on this so unfortunately we can't assign it to you..However, if they decide they're no longer working on it then we're happy to pass it to you. 

@charmilahruthika Do you have any updates on this?

**Comment by charmilahruthika** [comment]
Yes, I am working on it

**Comment by 4upz** [comment]
Awesome, sounds good @charmilahruthika! Feel free to continue! 

@hariomsinha7 We encourage you to take a look at other issues on our board, especially ones tagged for contributors to see where else you can contribute 😃

**Comment by charmilahruthika** [comment]
I am actually stuck, I am unable to run app on mocked data.

**Comment by 4upz** [comment]
@charmilahruthika Sorry to hear that. What type of error are you seeing after running the `yarn start-with-mock-data` command?

**Comment by charmilahruthika** [comment]
Proxy error: Could not proxy request /api/recommendations from localhost:3000 to http://localhost:4000.

**Comment by 4upz** [comment]
@charmilahruthika That's weird, it should be using a stub server when running with mock data. Sorry about that. I recommend trying one of the following:

- Refreshing the page or returning to the home page before trying to reload the recommendations page
- Quit the mock data process (ctrl + c), close the tab where the client loaded, and running the command again
- Make sure you're also on the latest version of our trunk branch, so that you're not catching weird bugs that we previously fixed.

Let me know if any of that helps. If not, it may be worth trying with real data if possible so that you aren't blocked and we can look into the issue separately

**Comment by charmilahruthika** [comment]
@4upz Thanks for the guidance. It worked. However the issue seems to have solved already. The horizontal scrolling bar is not visible, only the vertical scroll can be seen.

**Comment by 4upz** [comment]
@charmilahruthika Interesting. Have you tried with actual data? If not, you may need to hardcode custom data that reflect the image in the above behavior description as there is a chance that the mock data text is under the character limit in which this issue would occur

**Comment by charmilahruthika** [comment]
<img width="935" alt="Screenshot 2022-10-23 at 11 29 19 AM" src="https://user-images.githubusercontent.com/93112748/197376640-2be129e7-9981-473e-8cef-469fef023cf1.png">

@4upz the mock data actually have similar case as you see in this image

**Comment by 4upz** [comment]
Hi @charmilahruthika sorry for the late response. It seems like you're right! I just checked on my end and see that the text is also wrapping even with real data. Sorry for that confusion! This was probably inadvertently solved when the Emissions Dashboard was fixed. I'll go ahead and close this issue then, but feel free to take a look at our project board to see if there's another issue you'd like to work on!
